packages/sdks/output/qwik/vite.config.ts

```diff
@@ -11,19 +11,14 @@ export default defineConfig(() => {
         /**
          * https://github.com/BuilderIO/qwik/issues/4952
          */
-        fileName: (format) => `index.qwik.${format === 'es' ? 'mjs' : 'cjs'}`,
+        fileName: (format, entryName) => `${entryName}.qwik.${format === 'es' ? 'mjs' : 'cjs'}`,
       },
       rollupOptions: {
-        external: ['@builder.io/qwik', 'node:module'],
+        external: ['@builder.io/qwik', 'node:module', 'isolated-vm'],
+        input: ['./src/index.ts', './src/functions/evaluate/node-runtime/init.ts'],
```

Comment:
@sidmohanty11 just realized this means edge and browse bundles technically have `isolated-vm` imports.

We have to remove those to be safe and make sure no such imports exist in these two bundles